frontend/src/locales/fr.json

```diff
@@ -469,6 +469,8 @@
       "stakeLocked": "Stake locked",
       "warning": "Warning",
       "select": "Select",
+      "rewardsAvailableTooltip":"Rewards still available here",
```

Comment:
Just so you know for the future. i18n is implemented in a way that it will fallback to english if no value is provided in a selected language. Also, don't bother translating every single key to every language, as translators can do that. As a developer just add key-value to en.json and call it a day :D

Reply:
Ok, thank you for the tip :)